css/properties/transform-origin.json

```
@@ -122,6 +122,9 @@
         "three_value_syntax": {
           "__compat": {
             "description": "Three-value syntax",
+            "matches": {
+              "regex_value": "^([\\d\\w%-]+|calc\\(.+\\)) +([\\d\\w%-]+|calc\\(.+\\)) +([\\d\\w-]+|calc\\(.+\\))+$"
```

I think other types of whitespace (e.g. tabs) can also be used between values, so `\s+` might be better than ` +` for separation. Also, I think there's an errant `+` at the end of the regex.
```suggestion
              "regex_value": "^([\\d\\w%-]+|calc\\(.+\\))\s+([\\d\\w%-]+|calc\\(.+\\))\s+([\\d\\w-]+|calc\\(.+\\))$"
```